### DISCUSSION TITLE
Convert existing directory to super large directory?

### DISCUSSION THREAD

**Discussion by danthegoodman1** [seaweedfs/seaweedfs#5905] [Q&A] [answered]
Is this possible? It might be great to mention in the wiki

**Comment by chrislusf** [comment]
Yes and yes.

**Reply by danthegoodman1** [reply]
Great, is there any needed process than otherwise declaring it? I imagine it has to do some backfill?

**Reply by danthegoodman1** [reply]
just following up here @chrislusf ! sorry for so many discussions XD I'm very eager to learn more about SWFS

**Comment by chrislusf** [comment, marked as answer]
My previous answer is not accurate.

For Cassandra filer store, existing directory can not be converted to super large directory.
For Redis filer store, existing directory can be turned into super large directory, but this is not reversible.

**Reply by danthegoodman1** [reply]
Gotcha, for redis is it automatic?

**Reply by chrislusf** [reply]
correct
